src/app/core/services/material.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from "@angular/core";
+import { MatSnackBar } from "@angular/material/snack-bar";
+
+@Injectable({
+  providedIn: 'root',
+})
+export class MaterialService {
```

Comment:
This service is about Snackbar
So you don't need to create general material service and keep all the material data here
Better is to create SnackbarService that will contain snackbar related stuff